blockbase/block-templates/index.html

```diff
@@ -6,6 +6,7 @@
 	<!-- wp:post-title {"isLink":true} /-->
```

Comment:
KIND of unrelated to the change, but in order to get multiple items to show up to evaluate the spacer below I had to change this `<!-- wp:query-loop -->` to `<!-- wp:post-template -->` to get multiple items per-page.  This was a recent change to Gutenberg and would fit nicely into this change I think.

See [this change](https://github.com/Automattic/themes/pull/4107) for the same change in quadrat's templates.

Reply:
Ahh I did this see when I was trying to figure out the query loop changes. I wasn't sure which one was correct so I went with the current one. This makes sense, thanks for explaining!